app/services/invite_schools.rb

```diff
@@ -48,7 +48,7 @@ def send_nomination_email(recipient, token, school_name, email_expiry_date)
   end
 
   def recipient_email(school)
-    school.primary_contact_email || school.secondary_contact_email
+    school.primary_contact_email.presence || school.secondary_contact_email
```

Comment:
Should we prevent blank values from being written into the db?

Reply:
I guess the only place we are currently doing it is school importer, and the empty strings come from a quirk of how the `csv` gem handles empty values in a row.

I am not sure we want to handle it in both places, because it could lead to inconsistencies in the usage of the values - some places checking for `presence`, others relying on the earlier check.

Personally I think making the school importer insert `nil`s instead of empty strings in the values it doesn't receive would be nice to have, precisely because it would prevent accidents like this in other fields.

Reply:
Will add tech debt ticket